common/src/main/java/com/wynntils/features/overlays/CustomBarsOverlayFeature.java

```diff
@@ -1,237 +1,95 @@
-            Texture universalBarTexture = Texture.UNIVERSAL_BAR;
-
-            BufferedRenderUtils.drawColoredProgressBar(
-                    poseStack,
-                    bufferSource,
-                    universalBarTexture,
-                    this.textColor.get(),
-                    this.getRenderX(),
-                    renderY,
-                    this.getRenderX() + this.getWidth(),
-                    renderY + renderHeight,
-                    0,
-                    0,
-                    universalBarTexture.width(),
-                    universalBarTexture.height(),
-                    progress);
+        protected float getTextureHeight() {
+            return Texture.UNIVERSAL_BAR.height() / 2f;
         }
 
-        protected void renderText(
-                PoseStack poseStack, MultiBufferSource.BufferSource bufferSource, float renderY, String text) {
-            BufferedFontRenderer.getInstance()
-                    .renderAlignedTextInBox(
-                            poseStack,
-                            bufferSource,
-                            text,
-                            this.getRenderX(),
-                            this.getRenderX() + this.getWidth(),
-                            renderY,
-                            0,
-                            this.textColor.get(),
-                            this.getRenderHorizontalAlignment(),
-                            this.textShadow.get());
+        @Override
+        protected BarOverlayTemplatePair getActualPreviewTemplate() {
+            return new BarOverlayTemplatePair("3/10", "capped(3; 10)");
         }
     }
 
-    protected static class HealthBarOverlay extends BaseBarOverlay {
-        @RegisterConfig("feature.wynntils.customBarsOverlay.overlay.healthBar.healthTexture")
+    protected static class CustomHealthBarOverlay extends CustomBarOverlayBase {
+        @RegisterConfig("overlay.healthBar.healthTexture")
         public final Config<HealthTexture> healthTexture = new Config<>(HealthTexture.a);
 
-        protected HealthBarOverlay() {
-            this(
-                    new OverlayPosition(
-                            -30,
-                            -52,
-                            VerticalAlignment.Bottom,
-                            HorizontalAlignment.Center,
-                            OverlayPosition.AnchorSection.BottomMiddle),
-                    new OverlaySize(81, 21));
-        }
-
-        protected HealthBarOverlay(OverlayPosition overlayPosition, OverlaySize OverlaySize) {
-            super(overlayPosition, OverlaySize, CommonColors.RED);
+        public CustomHealthBarOverlay(int id) {
+            super(id, new OverlaySize(81, 21));
         }
 
         @Override
-        public float textureHeight() {
-            return healthTexture.get().getHeight();
+        public Texture getTexture() {
+            return Texture.HEALTH_BAR;
         }
 
         @Override
-        public String icon() {
```

Comment:
Do we lose the heart icon?

Reply:
All of this feature was moved. Diff is weird. Check description for more info

Reply:
Ok, I understand now.